worker/utils/environment.js

```diff
@@ -14,6 +15,15 @@ class EnvironmentClass {
     return jobDb;
   }
 
+  /* collection name is dynamic to allow mult staging autobuilder instances running
+     at same time without picking up each other's jobs */
+  static getCollection() {
```

Comment:
Does this need to be a static function, or could it just be a constant? It doesn't seem to be overridden AFAICS, and it looks like it will always return the same thing?

Reply:
If we forget to update the env vars in the script on one of our production sites for instance (such that jobCol === undefined), without this static function, the autobuilder will fail and there will be an outage. I think its smart to keep it as a static function in the interest of safety check

Reply:
For visibility/longevity for anyone in the future who may spy on this PR: I misunderstood Andrew's point, however it was agreed that it made more sense to keep with the established style of the codebase (even if its a java-ism) than to change how we retrieve one var (`const collection = (jobCol === undefined?) "queue" : jobCol;`).